reporters/templates/template.txt

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+</head>
+<body>
+    <p>{{ code }} ({{symbol}}) {{obj}}</p>
```

Comment:
@shweta97 I asked you to look into use a `for` loop in here, to only repeat the `<p>` part for each message, not the entire template.